pkg/cmd/loadflavor.go

```diff
@@ -121,9 +127,9 @@ func getPresignedURL(flavorName string) (string, error) {
 	return urlStr, nil
 }
 
-func genCommand(input string, pod *corev1.Pod) *exec.Cmd {
+func genCommand(input string, contextName string, pod *corev1.Pod) *exec.Cmd {
 
-	cmdArgs := []string{"exec", "-i", "-n", pod.Namespace, pod.Name, "--", "python", "manage.py", "loadflavor", input}
+	cmdArgs := []string{"exec", "-i", "-n", pod.Namespace, pod.Name, fmt.Sprintf("--context=%s", contextName), "--", "python", "manage.py", "loadflavor", input}
```

Comment:
I don't think you actually need the sprintf? Should work with `"--context", contextName` as separate args?